### ISSUE TITLE
Please offer a RESTful API in Meteor core for Meteor & non-Meteor clients

### ISSUE THREAD

**Issue by formspoint** [meteor/meteor-feature-requests#141]
I would like to use Meteor for a RESTful API.

My options:
[Restivus](https://github.com/kahmali/meteor-restivus) is abandoned with many open Issues and PRs and no commits or new development.

[Simple:rest](https://github.com/stubailo/meteor-rest) was last updated a year ago, many Issues & PRs pending, and looks like new new development. 

I read an article on adding [Express to Meteor](http://www.mhurwi.com/meteor-with-express/).

A feature like this would be helpful for mobile and enterprise systems. 

Any chance we could see something like this added to Meteor core?

**Comment by hwillson** [comment]
I don't think this needs to be part of core. While [nimble:restivus](https://atmospherejs.com/nimble/restivus) and [simple:rest](https://atmospherejs.com/simple/rest) may no longer be maintained, the fact that they exist demonstrates that Meteor already gives you all of the tools you need to provide a REST based API. Adding functionality like this to the core (that can already be built completely outside of the core)  increases the complexity of the Meteor codebase, which in turn increases its maintenance overhead. Anyone interested can fork those packages and start to maintain new versions, or you can consider tying `express` into Meteor (as you mentioned). That being said, if you're looking for something that's even more simple to use (and is already part of Meteor), you can build your REST API using [`connect`](https://github.com/senchalabs/connect). For an example of how this can be done, take a look at the source of [meteorhacks:picker](https://github.com/meteorhacks/picker) (which might be a bit more complex than you need, but the point is all of this functionality can be built using the tech already in Meteor).

**Comment by formspoint** [comment]
Here’s a lib that might be helpful to some: https://github.com/aadamsx/fine-rest
